src/components/header/KidsDropdown.jsx

```diff
@@ -0,0 +1,322 @@
+              <MenuBold2>Tights & Leggings</MenuBold2>
+              <MenuBold2>Jeans, Trousers & Capris</MenuBold2>
+              <MenuBold2>Jacket, Sweater & Sweatshirts</MenuBold2>
+              <MenuBold2>Innerwear & Thermals</MenuBold2>
+              <MenuBold2>Nightwear & Loungewear</MenuBold2>
+              <MenuBold2>Value Packs</MenuBold2>
+            </div>
+          </Grid>
+          <Grid item>
+            <div
+              style={{
+                backgroundColor: '#fff',
+                padding: '6px',
+                height: '440px',
+                width: '220px',
+              }}
+            >
+              <Heading>
+                <b>Footwear </b>
+              </Heading>
+              <MenuBold>Casual Shoes</MenuBold>
+              <MenuBold>Flipflops</MenuBold>
+              <MenuBold>Sports Shoes</MenuBold>
+              <MenuBold>Flats</MenuBold>
+              <MenuBold>Sandals</MenuBold>
+              <MenuBold>Heels</MenuBold>
+              <MenuBold>School Shoes</MenuBold>
+              <MenuBold>Socks</MenuBold>
+              <ColoredLine color="rgba(245,245,246,.4)" />
+              <Heading>
+                <b>Toys </b>
+              </Heading>
+              <MenuBold>Learning & Development</MenuBold>
+              <MenuBold>Activity Toys</MenuBold>
+              <MenuBold>Soft Toys</MenuBold>
+              <MenuBold>Action Figure / Play set</MenuBold>
+            </div>
+          </Grid>
+          <Grid item>
+            <div
+              style={{
+                backgroundColor: 'rgba(245,245,246,.4)',
+                padding: '6px',
+                height: '440px',
+                width: '220px',
+              }}
+            >
+              <Heading>
+                <b>Infants </b>
+              </Heading>
+              <MenuBold>Bodysuits</MenuBold>
+              <MenuBold>Rompers & Sleepsuits</MenuBold>
+              <MenuBold>Clothing Sets</MenuBold>
+              <MenuBold>Tshirts & Tops</MenuBold>
+              <MenuBold>Dresses</MenuBold>
+              <MenuBold>Bottom wear</MenuBold>
+              <MenuBold>Winter Wear</MenuBold>
+              <MenuBold>Innerwear & Sleepwear</MenuBold>
+              <MenuBold>Infant Care</MenuBold>
+              <ColoredLine color="rgba(245,245,246,.4)" />
+              <Heading2>
+                <b>Home & Bath </b>
+              </Heading2>
+              <Heading2>
+                <b>Personal Care </b>
+              </Heading2>
+            </div>
+          </Grid>
+          <Grid item>
+            <div
+              style={{
+                backgroundColor: '#fff',
+                padding: '6px',
+                height: '440px',
+                width: '220px',
+              }}
+            >
+              <Heading>
+                <b>Kids Accessories </b>
+              </Heading>
+              <MenuBold>Bags & Backpacks</MenuBold>
+              <MenuBold>Watches</MenuBold>
```

Comment:
These same codes where written at some class you can have a HOC for this so that passing an argument will give an UI aCCORDING TO the inputs. say Strings in <MenuBold>